src/main/java/com/airdnb/clone/domain/reservation/entity/Reservation.java

```diff
@@ -57,9 +58,14 @@ public class Reservation extends BaseTimeEntity {
     private Guest guest;
 
     @Builder.Default
+    @Enumerated(EnumType.STRING)
     @Column(name = "STATUS")
     private Status status = Status.PENDING;
 
+    public void updateGuest(Guest guest) {
```

Comment:
```suggestion
    public void changeGuest(Guest guest) {
```

`changeGuest` 로 바꾸지 않았었나요??